portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,12 +34,12 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /**
+  * Loads messages from database and transform them into a JSON object
+  */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    String stringNumberOfMessages = request.getParameter("nr");
-    int numberOfMessages = Integer.parseInt(stringNumberOfMessages);
-    
+    int numberOfMessages = Integer.parseInt(request.getParameter("nr"));
     if(numberOfMessages == 0){
```

Comment:
This seems counter-intuitive, if I ask for 0 messages why are you setting them to 100?

Also, the PR talks about login, why is this being modified here?

Reply:
I had no idea how to get all the comments.
In a later branch I've modified this.